apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -105,9 +103,17 @@ private void configureChannelBoxes(int channelCount) {
 
     public void closeAudio() {
         configureChannelBoxes(0);
+        mNativeApiSpinner.setEnabled(true);
```

Comment:
This stuff should be moved to stopAudio().

Reply:
And pauseAudio()

Reply:
Changed to all 3. Seems like we need it for pause, stop, and close.